bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WorkbenchPage.java

```diff
@@ -2826,14 +2828,17 @@ private void extendPerspectives(MPerspectiveStack perspectiveStack) {
 						desc, this, true);
 
 				PerspectiveExtensionReader reader = new PerspectiveExtensionReader();
-				reader.setIncludeOnlyTags(new String[] { IWorkbenchRegistryConstants.TAG_ACTION_SET });
+				reader.setIncludeOnlyTags(new String[] { IWorkbenchRegistryConstants.TAG_ACTION_SET,
+						IWorkbenchRegistryConstants.TAG_EDITOR_ONBOARDING_COMMAND });
```

Comment:
Why do you only add the "..command" here? what if e.g. on the text changes?

Reply:
Added text and image as well.
I needed to change PerspectiveExtensionReader.processExtension(IConfigurationElement element) for that.
Please have a look.

Tests will follow soon with a new PR, like discussed already.